utils/src/main/scala/za/co/absa/enceladus/utils/types/parsers/DecimalParser.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.types.parsers
+
+class DecimalParser(val precision: Int, val scale: Int) {
+
+  def parseDecimal(value: String): BigDecimal = {
+    val result = BigDecimal(value)
+    if (result.precision > precision || result.scale > scale) {
```

Comment:
Not sure about this. When I expect the scale to be 10. I expect exactly 10 numbers. This check only if it is bigger. Is that enough? What about smaller?